doc/api/cli.md

```diff
@@ -51,6 +51,28 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run [command]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+This runs an arbitrary command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+By default, this command adds the current path appended by
+`node_modules/.bin` directory to the PATH in order to execute the binaries
+from dependencies.
+
+For example, the following command will run the `test` script of
+the current project's `package.json`:
+
+```console
+$ node run test
+```
```

Comment:
Add examples of how the arguments are handled, like:

````suggestion
```console
$ node run test
```

You can also pass arguments to the script, any argument after the script name will be appended to the script:
```console
$ node run test --verbose
```
````